packages/dot-merlin-reader/dot-merlin-reader.3.5.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer:   "defree@gmail.com"
+authors:      "The Merlin team"
+synopsis:     "Reads config files for merlin"
+homepage:     "https://github.com/ocaml/merlin"
+bug-reports:  "https://github.com/ocaml/merlin/issues"
+dev-repo: "git+https://github.com/ocaml/merlin.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.02.1" & < "4.12"}
+  "dune" {>= "1.8.0"}
```

Comment:
```suggestion
  "dune" {>= "2.7.0"}
```
I noticed that upstream seems to be already fixed

Reply:
It is not fixed on every branch, I will change that.